### PR TITLE
feat(Bonus Pagamenti Digitali): [#175883276] Adds the SatispayCard Wallet preview

### PR DESCRIPTION
**Pull request by CrisTofani**
## Short description
This PR adds the SatispayWalletPreview component to walletv2 items list

### CODE REVIEW

File: ts/features/wallet/component/WalletV2PreviewCards.tsx [hunk 1]
```diff
@@ -30,13 +34,24 @@ const WalletV2PreviewCards: React.FunctionComponent<Props> = props => (
       )),
       null
     )}
+    {pot.getOrElse(
```
Comment (Undermaken) [inline ts/features/wallet/component/WalletV2PreviewCards.tsx:37]:
```suggestion
    {pot.toUndefined(
      pot.mapNullable(props.satispayList, s => (
        <>
          {s.map(s => (
            <SatispayWalletPreview key={s.idWallet} />
          ))}
        </>
      ))
    )}
```
could it be a valid refactoring?

File: ts/features/wallet/satispay/SatispayWalletPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ */
+const SatispayWalletPreview: React.FunctionComponent<Props> = props => (
+  <CardPreview
+    left={
+      <Body style={IOStyles.flex} numberOfLines={1}>
+        {I18n.t("wallet.methods.satispay.name")}
+      </Body>
+    }
+    image={satispayImage}
+    onPress={() => props.navigateToSatispayDetails()}
```
Comment (Undermaken) [inline ts/features/wallet/satispay/SatispayWalletPreview.tsx:28]:
```suggestion
    onPress={props.navigateToSatispayDetails}
```

File: ts/features/wallet/satispay/SatispayWalletPreview.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  />
+);
+
+const mapDispatchToProps = (_: Dispatch) => ({
+  // TODO replace with correct navigation action
+  navigateToSatispayDetails: () => nullType
```
Comment (Undermaken) [inline ts/features/wallet/satispay/SatispayWalletPreview.tsx:34]:
```suggestion
  navigateToSatispayDetails: constNull
```
`import { constNull } from "fp-ts/lib/function";`
it is shorter ;)

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/features/wallet/satispay/SatispayWalletPreview.tsx (hunk 2 above)
Comment (CrisTofani) [reply ts/features/wallet/satispay/SatispayWalletPreview.tsx:28]: @Undermaken I would accept the suggestion, but this is going to be replaced by the following implementation of correct navigation action see [here](https://github.com/pagopa/io-app/pull/2458/files#diff-d4a25fe5860c665ade63a83a9b2717c0a07c2730801becbbb220b635f472b865R34) 😄

File: ts/features/wallet/satispay/SatispayWalletPreview.tsx (hunk 3 above)
Comment (CrisTofani) [reply ts/features/wallet/satispay/SatispayWalletPreview.tsx:34]: @Undermaken same as previous comment but I'll keep this in mind for future purposes of course 😄

File: ts/features/wallet/component/WalletV2PreviewCards.tsx (hunk 1 above)
Comment (CrisTofani) [reply ts/features/wallet/component/WalletV2PreviewCards.tsx:37]: Yes sure! sounds good!

File: ts/features/wallet/satispay/SatispayWalletPreview.tsx (hunk 2 above)
Comment (Undermaken) [reply ts/features/wallet/satispay/SatispayWalletPreview.tsx:28]: No problem ;)
Leave it as it is

**Review by Undermaken — APPROVED** [review]

**Pull request merged**